pkg/apis/acid.zalan.do/v1/operator_configuration_type.go

```diff
@@ -242,6 +242,7 @@ type OperatorConfigurationData struct {
 	MajorVersionUpgrade        MajorVersionUpgradeConfiguration   `json:"major_version_upgrade"`
 	Kubernetes                 KubernetesMetaConfiguration        `json:"kubernetes"`
 	PostgresPodResources       PostgresPodResourcesDefaults       `json:"postgres_pod_resources"`
+	ReadOnlyRootFilesystem     *bool                              `json:"read_only_root_filesystem,omitempty"`
```

Comment:
please move it into the KubernetesMetaConfiguration struct next to spiloPrivilege option